changelog/bug-1591236.md

```diff
@@ -0,0 +1,4 @@
+level: minor
+reference: bug 1591236
+---
+Linking Docs pages to their source codes.
```

Comment:
The [best practices](https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/changelog.md) for our changelogs suggest:
 * `silent`: Changes to tests or documentation
 * Write it in the present tense, as if the change had just been deployed.

Since it's silent, there's no need for text here, but the second bullet point would mean something like "Documentation pages now contain links to their source code".

Reply:
@djmitche You are right, giving the docs link in the status bar would make more sense. But wont this result in two nav-bar components in `ui/src` .

Reply:
I don't think it needs another navbar component -- just different placement.